compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/UIKitTextInputService.uikit.kt

```diff
@@ -127,7 +130,14 @@ internal class UIKitTextInputService(
         currentImeOptions = null
         currentImeActionHandler = null
         hideSoftwareKeyboard()
-        textUIView?.removeFromSuperview()
+
+        textUIView?.input = null
+        textUIView?.let { view ->
+
+            dispatch_async(dispatch_get_main_queue()) {
```

Comment:
@dima-avdeev-jb , do you have something in you mind how to get similar effect using coroutines?

Reply:
In coroutines we have `yield()` function

Reply:
Done!